app/main.development.js

```diff
@@ -428,6 +428,10 @@ function createDexWindow(serverAddress) {
   dexWindow.once("ready-to-show", () => {
     dexWindow.show();
   });
+  dexWindow.webContents.on("will-navigate", (e, url) => {
+    mainWindow.webContents.send("open-external", url);
+    e.preventDefault();
+  });
```

Comment:
So, I noticed that not especially the `https://tbtc.bitaps.com/*` links displayed with a white screen, but links targeted with `_blank`. The `will-navigate `works great with `_self` links, but for `_blank` links I suggest this event handler additionally:
```
  dexWindow.webContents.setWindowOpenHandler(({ url }) => {
    mainWindow.webContents.send("open-external", url);
    return { action: "deny" };
  });
```


<img width="702" alt="2022-05-13_17-46" src="https://user-images.githubusercontent.com/52497040/168320038-f2fda68b-7263-4264-8e09-7c626a1e01dd.png">

Reply:
Just tested and this suggestion works, @bgptr.

The other link was changed to `_blank` in 0.5 BTW, but might as well  handle both targets.

Would be create to get this PR in for the 0.5 DEX update (https://github.com/decred/decrediton/pull/3791).

Reply:
Having tested more, I think the safest thing is to open an external browser for **only** `_blank` target links.  All of the explorer links are now `_blank` targets.  I'm concerned about some normal app navigation clicks going to an external window instead.  I've only seen that happen with "Sign Out" so far though.

Reply:
Yeah, in https://github.com/decred/decrediton/pull/3791 I'm using just `setWindowOpenHandler` and **not** `on("will-navigate", ...)` and it's giving the desired effect.  I've put these commits on that PR branch for testing.

Can we either get this PR revised and merged, or closed and just use the commits included in https://github.com/decred/decrediton/pull/3791?